README.md

```diff
@@ -654,6 +657,32 @@ An example response for all `POST` commands:
 }
 ```
 
+### Send POI
+
+Sending a POI to the car.
+
+* `POST /webapi/v1/user/vehicles/:VIN/snedpoi`
+    * Where `:VIN` is your vehicle's VIN.
+    * Remember to include the `Authorization: Bearer` header.
+    * Remember to include the `Content-Type: application/x-www-form-urlencode` header.
+
+#### Body
+```
+body={
+	"poi": {
+		"city": "abc",
+		"lat": nn.nnn,
+		"lon": nn.nnn,
+		"name": "name",
+		"rating": -1,
+		"postalCode: 12345,
```

Comment:
I think this should be `"postalCode": "12234",` - not all countries have numeric post codes. Also, note the closing `"` on `"postalCode"`